android/app/src/main/res/layout/activity_manual_control.xml

```diff
@@ -109,4 +109,20 @@
         app:layout_constraintTop_toBottomOf="@+id/joystickView2"
         app:layout_constraintVertical_bias="0.428" />
 
+    <Button
+        android:id="@+id/button2"
```

Comment:
Renaming the button ID to something like autonomousDrivingButton for example, may improve readability/maintainability of the code.